app.js

```diff
@@ -19,7 +19,7 @@ app.use(bodyParser.urlencoded({
   extended: true
 }));
 app.use(session({
-  secret: process.env.SECRET,
+  secret: "test",
```

Comment:
Should leave secret as the env file SECRET

Reply:
I don't know how this got here

Reply:
https://github.com/Shoreline-CS-Club-2021-2022/website/commit/e1009d67458fc1a885353a35cd1eb83f676fd37c apparently. I am very confused.

Reply:
I'll fix it

Reply:
Ok, fixed.